FakeTrello/Models/CardList.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class CardList
+    {
```

Comment:
If the intention for this class is to represent the one-to-many relationship between Lists and Cards, you don't need to make a class to represent a join table in SQL. Entity Framework will create the join table without this class with just the List<Card> in your List class.